tests/integration/targets/ios_vlans/tasks/main.yaml

```diff
@@ -3,6 +3,20 @@
   ansible.builtin.set_fact:
     skip_test: false
 
+- name: Gather device information
+  ansible.builtin.ios_command:
+    commands:
+      - show version
+  register: show_version_output
+
+- name: Identify if device is Catalyst 9000
+  ansible.builtin.set_fact:
+    is_cat9k: "{{ 'CAT9K_IOSXE' in show_version_output.stdout | join(' ') }}"
```

Comment:
we should try to check l2 and l3 mostly and not add conditionals to any specific version.

Reply:
added, can you re-review again?